tensorflow/tools/tf_windows_build_dockerfiles/README.md

```diff
@@ -0,0 +1,23 @@
+**To build a Windows container locally, execute these commands. These commands have been tested on Windows PowerShell when run as an administrator:**
+```
+md C:\tf
+md C:\tf\tmp
+cd C:\tf
+git clone https://github.com/Bobarshad/tensorflow.git
+cd C:\tf\tensorflow\tensorflow\tools\tf_windows_build_dockerfiles\
+docker build . -f <Dockerfile 3.9 or 3.11> -t win-docker
+```
+
+**Starting docker:**
+```
+docker run --name tf -itd --rm -v C:\tf\tensorflow:C:\workspace -v C:\tf\tmp:C:\tmp -e TEST_TMPDIR=C:\tmp -w C:\workspace win-docker bash
+```
+
+**To build TensorFlow pip_package:**
```

Comment:
This seems to indicate that the containers are only useful for building TensorFlow. Am I mistaken in that the goal for these containers is to replace the complete Windows build+test environment (CPU: build + test;  GPU: just build) similar to the way the newer Linux containers did? You'll need to make sure they can also replicate our current Windows test suites, plus the libtensorflow builds.

Reply:
It is changed.